ingestion_tools/pyproject.toml

```diff
@@ -46,8 +46,7 @@ boto3-stubs = {extras = ["s3"], version = "^1.34.34"}
 mypy = "^1.8.0"
 cryoet-data-portal-neuroglancer = { git = "https://github.com/chanzuckerberg/cryoet-data-portal-neuroglancer.git", branch = "main" }
 distinctipy = "^1.3.4"
-imodmodel = ">=0.0.12"
-linkml = "^1.8.1"
+imodmodel = "0.0.12"
```

Comment:
imodmodel = "0.0.13" has breaking changes, (see the rename of slan to slicer_angle), so as per @jgadling, we will stick to 0.0.12 for now. 

See https://github.com/teamtomo/imodmodel/commit/5908efc1993ff076bc6196ac992a5bf20476c4c3#diff-5073310c046e4db3049d5d7f98bf4fa26775dd55aab3ec4c4d7e4defb780178e